sdk/tables/Microsoft.Azure.WebJobs.Extensions.Tables/src/PocoToTableEntityConverter.cs

```diff
@@ -17,7 +17,7 @@ internal class PocoToTableEntityConverter<TInput>: IConverter<TInput, TableEntit
 
         public PocoToTableEntityConverter()
         {
-            _info = PocoTypeBinder.Shared.GetBinderInfo(typeof(TInput));
+            _info = PocoTypeBinder.Shared.GetBinderInfo(typeof(TInput), typeof(ITableEntity));
```

Comment:
Don't we need to support both interface and non-interface? Is this a breaking change?

Reply:
Or I suppose we are doing that here? If so, can we remove the other GetBinderInfo method and ctor?

Reply:
What's the scenario you have in mind for non-interface?

Reply:
In the extension bindings I don't think we require that POCOs implement ITableEntity. We should have tests for this.

Reply:
It looks like we have a compile-time constraint for the Tables use case, so it is not a breaking change there.

Reply:
It's definitely a behavior change, but I'm not sure if I would call it a breaking change since serialization just wouldn't work properly before when explicitly implemented ITableEntity properties. Do you think it would make more sense to throw behind an option?

Reply:
The part that is breaking is if the type you are binding to did not implement ITableEntity at all.

Reply:
Looking at this again, after seeing the failed test, I don't think this converter needs the explicit interface support, since it is specifically to handle a scenario where the POCO does not implement `ITableEntity`

Reply:
> The part that is breaking is if the type you are binding to did not implement ITableEntity at all.

That is currently not supported, at least in Tables proper. I'll do some more investigation for the extension scenarios.